src/ResourceManager/Advisor/Commands.Advisor/help/Enable-AzureRmAdvisorRecommendation.md

```diff
@@ -0,0 +1,162 @@
+---
+external help file: Microsoft.Azure.Commands.Advisor.dll-Help.xml
+Module Name: Az.Advisor
+online version:
```

Comment:
all of these need online versions. See other help files for help with the format.